src/sentry/api/endpoints/monitor_checkins.py

```diff
@@ -137,9 +146,22 @@ def post(
         result = serializer.validated_data
 
         with transaction.atomic():
+            environment = Environment.get_or_create(
+                project=project, name=result.get("environment", "production")
+            )
+
+            monitor_environment, created = MonitorEnvironment.objects.get_or_create(
+                monitor=monitor, environment=environment
+            )
+
+            if created:
+                monitor_environment.status = monitor.status
+                monitor_environment.save()
+
             checkin = MonitorCheckIn.objects.create(
                 project_id=project.id,
                 monitor_id=monitor.id,
+                monitor_environment_id=monitor_environment.id,
```

Comment:
```suggestion
                monitor_environment=monitor_environment,
```

Any reason not to write it like this?

Reply:
just for consistency with above. otherwise no reason